moduleroot/.github/workflows/ci.yml.erb

```diff
@@ -3,6 +3,10 @@ name: CI
 
 on:
   pull_request:
+  push:
+    branches:
+      - 'master'
```

Comment:
for vox pupuli, we also added `main`, just in case we ever rename a branch. We just wanted to be sure to not miss the CI config if/when we rename.

Reply:
that's fair!

Reply:
(even tho for Foreman it probably should be `develop` to match the rest of the project)